VERSION

```diff
@@ -0,0 +1,4 @@
+VERSION="2.0.0"
+VERSION_SUFFIX=-alpha1
```

Comment:
I think you should use
```suggestion
VERSION_SUFFIX=-alpha.1
```
If we want to follow  https://semver.org/#spec-item-9

Reply:
Thanks!

this review is very appreciated :')